task/distributed_queue.py

```diff
@@ -65,6 +66,15 @@
 }
 
 
+class PikaMethod:
+    message_count: int
+
+
+class DeclareResult:
+    # this is pika.spec.Basic.DeclareOk but that isn't exported
```

Comment:
I think this is `pika.spec.Queue.DeclareOk`, no?

Reply:
Ah, right! Got confused by https://pika.readthedocs.io/en/stable/_modules/pika/spec.html#Queue.DeclareOk . Queue.* works, thanks!